src/main/groovy/org/edx/jenkins/dsl/UserRetirementConstants.groovy

```diff
@@ -0,0 +1,112 @@
+/*
```

Comment:
The bulk of the DSL closures have moved into this shared class so that we don't have to re-type them for every single job.